tests/integration/checkpoint.bats

```diff
@@ -159,6 +159,17 @@ function simple_cr() {
 	# checkpoint the running container
 	mkdir image-dir
 	mkdir work-dir
+
+	# runc will fail whith absolute parent image path.
+	runc --criu "$CRIU" checkpoint --parent-path "$(pwd)"/parent-dir --work-path ./work-dir --image-path ./image-dir test_busybox
+	[[ "${output}" == *"--parent-path needs to be relative path of --image-path, and points to previous checkpoint directory"* ]]
+	[ "$status" -ne 0 ]
+
+	# runc will fail whith invalid parent image path.
+	runc --criu "$CRIU" checkpoint --parent-path ./parent-dir --work-path ./work-dir --image-path ./image-dir test_busybox
+	[[ "${output}" == *"--parent-path needs to be relative path of --image-path, and points to previous checkpoint directory"* ]]
+	[ "$status" -ne 0 ]
+
```

Comment:
simple_cr() is called from many places, I think this deserves a separate test case. Also, I guess it's enough to check that runc fails, and the error contains "--parent-path", rather than checking for a full specific error message (which might change).

Reply:
@kolyshkin Thanks very much for you review messages～.  I add a test case  to  test  `--parent-path`. Am I missing something？